ddtrace/contrib/pymongo/patch.py

```diff
@@ -71,6 +70,7 @@ def traced_get_socket(wrapped, instance, args, kwargs):
         "pymongo.get_socket", service=trace_utils.int_service(pin, config.pymongo), span_type=SpanTypes.MONGODB
     ) as span:
         span.set_tag_str(COMPONENT, config.pymongo.integration_name)
+        span.set_tag_str(db.SYSTEM, config.pymongo.dbms_name)
```

Comment:
add `from ddtrace.ext import mongo` to the top of the file

```suggestion
        span.set_tag_str(db.SYSTEM, mongo.SERVICE)
```